packages/owl-bot-post-processor/test/owl-core.ts

```diff
@@ -155,4 +157,81 @@ describe('core', () => {
       assert.strictEqual(build.summary, '1 steps failed 🙁');
     });
   });
+  describe('getOwlBotLock', () => {
+    it('reads .OwlBot.lock.yaml and returns parsed YAML', async () => {
+      const prData = {
+        data: {
+          head: {
+            ref: 'my-feature-branch',
+            repo: {
+              full_name: 'bcoe/example',
+            },
+          },
+        },
+      };
+      const config = `docker:
+  image: node
+  digest: sha256:9205bb385656cd196f5303b03983282c95c2dfab041d275465c525b501574e5c`;
+      const content = {
+        data: {
+          content: Buffer.from(config, 'utf8').toString('base64'),
+          encoding: 'base64',
+        },
+      };
+      const octokit = ({
+        pulls: {
+          get() {
+            return prData;
+          },
+        },
+        repos: {
+          getContent() {
+            return content;
+          },
+        },
+      } as any) as InstanceType<typeof Octokit>;
```

Comment:
`any` is currently turned off a the top of the file, because it was slightly painful turning it off for each instance, when mocking.

I don't know that the `any` check buys that much in unit tests? I 100% agree with the rule in code files.

Reply:
I have no qualms about casting to any in this case.

Reply:
I went ahead and made all of the `any` exceptions explicit, rather than having it at the top of a file.